style.css

```diff
@@ -1,825 +1,810 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+	box-sizing: inherit;
+}
+
 body {
-  margin: 0;
-  font-family: Montserrat, sans-serif;
+	min-width: 320px;
+	overflow-x: hidden;
+	font-family: Montserrat;
+	margin: 0;
 }
 
-.collection__img-mob {
-  display: none;
+h1 {
+	margin: 0;
+}
+
+.second__bag-dsk,
+.third-count,
+.head__desctop,
+.second__quantity,
+.second__other,
+.second__vector,
+.third__img-dsk {
+	display: none;
 }
 
 .header {
-  position: absolute;
-  z-index: 3;
-  width: 100%;
+	background-color: #fff;
 }
 
 .header__inner {
-  display: flex;
-  justify-content: space-between;
+	height: 68px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.header__inner-logo-link {
+	margin-left: 26px;
 }
 
-.header__logo {
-  font-size: 30px;
-  letter-spacing: 3px;
-  margin: 6px 52px;
-  z-index: 2;
-  color: #fff;
+.header__inner-menu-btn {
+	width: 70px;
+  height: 70px;
+  background-color: #ededed;
+  border: none;
+  position: relative;
 }
 
-.header__menu {
-  display: flex;
-  flex-direction: column;
-  height: 92px;
-  width: 92px;
-  justify-content: center;
-  align-items: center;
-  background: rgb(0, 0, 0, .05);
+.header__inner-menu-btn::after {
+  content: '';
+  position: absolute;
+  width: 26.4px;
+  height: 2.61px;
+  background-color: #000;
+  top: 29.66px;
+  left: 21.49px;
+  box-shadow: 0 7.08px;
 }
 
-.header__menu-item {
-  height: 3.48px;
-  width: 34.7px;
-  background: black;
-  margin-bottom: 5.96px;
+.head {
+	background: url(img/bg-head.png) no-repeat;
+	background-position: center;
+	background-size: cover;
+	min-height: 522px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)